common/src/main/java/com/wynntils/functions/HorseFunctions.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.functions;
+
+import com.wynntils.core.functions.ActiveFunction;
+import com.wynntils.core.managers.Model;
+import com.wynntils.mc.utils.InventoryUtils;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.wc.custom.item.ItemStackTransformModel;
+import com.wynntils.wc.custom.item.WynnItemStack;
+import com.wynntils.wc.custom.item.properties.ItemProperty;
+import java.util.List;
+import net.minecraft.world.item.ItemStack;
+
+public class HorseFunctions {
+
+    private static WynnItemStack getHorse() {
+        int horseSlot = InventoryUtils.findHorseSlotNum();
+        if (horseSlot == -1) return null;
+        ItemStack is = McUtils.player().getInventory().getItem(horseSlot);
+        if (!(is instanceof WynnItemStack horse)) return null;
+        return horse;
+    }
+
+    public static class HorseLevelFunction extends ActiveFunction<String> {
+        @Override
+        public String getValue(String argument) {
+            WynnItemStack horse = getHorse();
+            if (horse == null) return "??";
```

Comment:
Can we add a generic UNKNOWN string const (for every function)

Reply:
That is already present, and it is `null`. This is interpreted as a unknown/invalid (currently represented by `N/A` in the UI).

Reply:
We return "??" though?

Reply:
Yes, currently in the PR. I suggested above to @DonkeyBlaster that he should return `null` instead. I'm guessing he hasn't been around to it yet.

Reply:
I mentioned [here](https://github.com/Wynntils/Artemis/pull/316#discussion_r945554495) that having N/A in those two specific functions may look weird, I can change it though if you don't think it will be an issue.

Reply:
It is probably better then to replace the "N/A" value in `FunctionManager.getSimpleValueString()` to `??`, and return null here. I agree that it is probably more readable to have the "??" than the "N/A" string.